### PR TITLE
add horse.js

### PR DESCRIPTION
**Pull request by NguyenDuyXCXCXCXCXC**
check line to line

### CODE REVIEW

File: Horse.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function Horse(){
+
```
Comment (nguyenduyxc) [inline Horse.js:2]: xoa dong nay di

**Review by nguyenduyxc — COMMENTED** [review]
no comment

File: Horse.js (hunk 1 above)
Comment (NguyenDuyXCXCXCXCXC) [reply Horse.js:2]: oke nhe

File: Horse.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+}
+module.exports=Horse;
```
Comment (nguyenduyxc) [inline Horse.js:4]: nhin cai nay van chua on dau

Comment (nguyenduyxc) [inline Horse.js:4]: chua o k lam

**Review by nguyenduyxc — COMMENTED** [review]
nho sua cho ky nhe

**Pull request merged**